yarn.lock

```diff
@@ -0,0 +1,4280 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Previously, `npm-lock.json` is not generated and committed, so may it should be added into `.gitignore`?